lib/features/children/overview/widgets/profile_overview_tile.dart

```diff
@@ -0,0 +1,142 @@
+          backgroundColor: _getBackgroundColor(context),
+        ),
+        onPressed: () => _onTap(context, isGivtAccount),
+        child: SizedBox(
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            mainAxisSize: MainAxisSize.min,
+            children: [
+              ClipRRect(
+                borderRadius: BorderRadius.circular(10),
+                child: SvgPicture.network(
+                  profile.pictureURL,
+                  width: 64,
+                  height: 64,
+                ),
+              ),
+              const SizedBox(height: 8),
+              Text(
+                profile.firstName,
+                maxLines: 1,
+                overflow: TextOverflow.ellipsis,
+                textAlign: TextAlign.center,
+                style: Theme.of(context).textTheme.titleMedium?.copyWith(
+                      fontFamily: 'Raleway',
+                      fontWeight: FontWeight.w700,
+                      color: AppTheme.givtBlue,
+                    ),
+              ),
+              if (profile.isChild)
+                Padding(
+                  padding: const EdgeInsets.only(top: 2),
+                  child: Text(
+                    '$currencySymbol${profile.wallet.balance.toStringAsFixed(0)}',
+                    textAlign: TextAlign.center,
+                    style: Theme.of(context).textTheme.titleMedium?.copyWith(
+                          color: AppTheme.givtBlue,
+                        ),
+                  ),
+                ),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+
+  void _onTap(BuildContext context, bool isGivtAccount) {
+    if (profile.isAdult) {
+      AnalyticsHelper.logEvent(
+        eventName: AmplitudeEvents.adultProfileTileClicked,
+        eventProperties: {
+          'name': profile.firstName,
+        },
+      );
+      if (isGivtAccount) {
+        context.pushNamed(
+          Pages.personalInfoEdit.name,
+        );
+      }
+      return;
+    }
+    AnalyticsHelper.logEvent(
+      eventName: AmplitudeEvents.childProfileClicked,
+      eventProperties: {
+        'name': profile.firstName,
+      },
+    );
+
+    context.pushNamed(
+      Pages.childDetails.name,
+      extra: [
+        context.read<FamilyOverviewCubit>(),
+        profile,
+      ],
+    );
+  }
+
+  Color _getBackgroundColor(BuildContext context) {
+    if (profile.isAdult) {
+      return AppTheme.givtLightBackgroundBlue;
+    }
+    if (profile.isChild) {
+      return AppTheme.givtLightBackgroundGreen;
+    }
+    return Theme.of(context).colorScheme.error;
+  }
+
+  Color _getBorderColor(BuildContext context, bool isGivtAccount) {
+    if (isGivtAccount) {
+      return AppTheme.givtBorderBlue;
+    }
+    if (profile.isAdult) {
+      return AppTheme.givtDisabledBorderBlue;
+    }
+    if (profile.isChild) {
+      return AppTheme.givtLightGreen;
+    }
+    return Theme.of(context).colorScheme.onError;
+  }
+}
```

Comment:
nice and clear Widget!

Reply:
Its just a more generalised version of the children_overview_item, so kudos to you too 😄